hbase-connectors-assembly/pom.xml

```diff
@@ -1,6 +1,5 @@
-<?xml version="1.0"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
i'm fan of the old version, in two lines.

Reply:
can fix it in HBASE-28007

Reply:
Sure but this is how our current settings for pom formatting does (See in hbase repo: https://github.com/apache/hbase/blob/ab4b1d884f13d082a0906c3c3e9ec96bce7f3717/pom.xml#L1)

I have not tried to add any delta change in settings for java and scripts formatting and aligned the with hbase here?

Is it fine to take this as another change, across hbase project, where we tweak these settings globally?

Reply:
ok then, just follow `hbase`